src/splitInteger.test.js

```diff
@@ -8,12 +8,36 @@ describe(`Function 'splitInteger':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(splitInteger(8, 1)).toBeInstanceOf(Array);
   });
 
   it(`should return an array with 'num' if parts = 1`, () => {
+    expect(splitInteger(8, 1)).toEqual([8]);
+  });
+
+  it(`should return an array with 'num' if parts = 5`, () => {
+    expect(splitInteger(32, 5).length).toBe(5);
+  });
```

Comment:
According to your description, it should return an array `[32]`, because num = 32